R/ScpModel-Class.R

```diff
@@ -168,10 +168,26 @@ scpModelFilterNPRatio <- function(object, name, filtered = TRUE) {
 ##' @export
 scpModelResiduals <- function(object, name, join = TRUE,
```

Comment:
Will need to be rewrited when switching to msqrob2 modelling, we will be able to use a single matrix computation for all residuals. This is not possible in the current implementation since each linear model has his unique design matrix.